### PR TITLE
18.12 release

### PR DESCRIPTION
**Pull request by mkarmona**

### CODE REVIEW

**Comment by richardbrunt-tessella** [comment]
I think there might be a slight mistake here: https://github.com/opentargets/data_pipeline/blob/9f327c6088fdc713deedaf71c9ddced177a879b8/mrtarget/modules/ECO.py#L49

It currently reads

```python
        uri_so = Config.ONTOLOGY_CONFIG.get('uris', 'eco')
        uri_eco = Config.ONTOLOGY_CONFIG.get('uris', 'eco')
```

But I would have thought it should read:

```python
        uri_so = Config.ONTOLOGY_CONFIG.get('uris', 'so') # This line changed
        uri_eco = Config.ONTOLOGY_CONFIG.get('uris', 'eco')
```

**Comment by afaulconbridge** [comment]
Good catch @richardbrunt-tessella ! After internal discussion, we're going to leave it in here for now since otherwise we'd have to go back and re-do the data release process, and this gives us a test if it is actually used anywhere. I've created opentargets/data_pipeline#397 to fix it in master for 19.02

File: mrtarget/modules/EvidenceString.py [hunk 1]
```diff
@@ -403,7 +377,11 @@ def check_is_valid_evs(self, evidence, datasource):
         if efo_id not in self.available_efos:
             problem_str = "%s Evidence %s has an invalid efo id in disease.id: %s" % (datasource, evidence_id, efo_id)
             return False, problem_str
-
+        if self.is_excluded_by_biotype(datasource, gene_id):
+            problem_str = problem_str = "%s Evidence %s gene_id %s is an excluded biotype" % \
```
Comment (afaulconbridge) [inline mrtarget/modules/EvidenceString.py]: it works, but... !

Comment (mkarmona) [reply mrtarget/modules/EvidenceString.py]: @afaulconbridge fixed!

**Pull request merged**